### ISSUE TITLE
Module displaying all games even with focus_on config option.

### ISSUE THREAD

**Issue by danhuss** [fewieden/MMM-NFL#55]
Platform (Hardware/OS): Raspberry Pi/Raspbian

Node version: ?

MagicMirror version: latest stable

Module version: latest

Description of the issue: All of the scheduled games are displaying even when using the "focus_on" config option. This has worked in the past and seems like it's not in this upcoming season. I'm not sure when exactly it started happening.

Thanks,
Daniel

**Comment by fewieden** [comment]
Hi @danhuss, do you mind sharing your config for MMM-NFL? Last season I switched out the data source, maybe something broke there, or maybe it is related to the pre-season.

I won't judge you for your favorite team :stuck_out_tongue_winking_eye:

**Comment by danhuss** [comment]
Panthers all the way... I do just like to keep an eye on the rest of the division as well.

                {
                        module: "MMM-NFL",
                        position: "bottom_left",
                        config: {
                                focus_on: ["CAR", "ATL", "NO", "TB"],
                                colored: "true"
                        }
                },

**Comment by fewieden** [comment]
@danhuss the match list wasn't filtered anymore at all based on focus_on, it only added the teams to the list if they were on bye week. I'm filtering now also the match list.

You can check out the work in progress on the develop branch, since I want to make some more adjustments to the module before releasing a new version.

```
cd ~/MagicMirror/modules/MMM-NFL
git checkout develop
git pull
npm ci
```

Also a note to your config option `colored`. The value should be a boolean `true` or `false`. You wrapped it in quotes which changes the type to a string. A string always get's treated as `true`, except when the string is empty. In your case, it works because you want it to be `true`, but if you change it for a test to `"false"` you will see that it still gets treated as `true`. I suggest changing it, this might cause you trouble when configuring other modules and you wrap booleans in quotes.

**Comment by fewieden** [comment]
@danhuss I already did the other stuff :smile: I just released version 1.3.1 which contains the fix, so you only have to update the module, no need to switch to the develop branch
